requirements/edx/github.txt

```diff
@@ -74,8 +74,7 @@ git+https://github.com/edx/lettuce.git@0.2.20.002#egg=lettuce==0.2.20.002
 -e git+https://github.com/edx/event-tracking.git@0.2.1#egg=event-tracking==0.2.1
 -e git+https://github.com/edx/django-splash.git@v0.2#egg=django-splash==0.2
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
-git+https://github.com/stvstnfrd/edx-ora2.git@hawthorn/wip#egg=ora2
--e git+https://github.com/edx/edx-submissions.git@2.0.0#egg=edx-submissions==2.0.0
```

Comment:
Nice catch!

Looks like edx is now pinning this to `2.1.1` on PyPi (finally "off" github) in `requirements/edx/base.in`.